CMakeLists.txt

```
@@ -61,8 +61,15 @@ option(BUILD_PYTHON_BINDINGS "Build Python bindings with pybind11." OFF)
 option(BUILD_TESTING_PYTHON "Build Python tests only." OFF)
 
 if (BUILD_PYTHON_BINDINGS)
+
+  # Include the vendored FindPython3 resources
+  list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
+
+  find_package(Python3 COMPONENTS Interpreter Development.Module)
```

I think you should use `REQUIRED` in `find_package`: 
"The REQUIRED option stops processing with an error message if the package cannot be found."

```suggestion
  find_package(Python3 REQUIRED COMPONENTS Interpreter Development.Module)
```

---

Good catch, thanks! I copied and pasted this line from the previous logic that didn't require REQUIRED and then forgot to update it.

---

I commited and squashed the suggestion, I forgot to get the commit before force pushing but it is now included in 2aea66e.

---

Done in adf6fe6